### ISSUE TITLE
keyword case with FOR ALL ENTRIES

### ISSUE THREAD

**Issue by OberstVonGatow** [abaplint/abaplint#2112]
It looks like the keyword case rule gives me a false positive on Selects with FOR ALL ENTRIES:

```
    SELECT FROM vekp
    FIELDS exidv
    FOR ALL ENTRIES IN @lt_hus
    WHERE vekp~exidv = @lt_hus-package_no
    INTO TABLE @DATA(lt_sub_vekp).
```

abaplint Message:
"Identifiers should be lower case: "FOR" (keyword_case)"

I also don't understand why excluding FOR in the rules doesn't have any effect (I also tried "^FOR"):
```
        "keyword_case": {
        "exclude": [],
        "severity": "Error",
        "style": "upper",
        "ignoreExceptions": true,
        "ignoreLowerClassImplmentationStatement": true,
        "ignoreGlobalClassDefinition": false,
        "ignoreGlobalInterface": false,
        "ignoreFunctionModuleName": false,
        "ignoreGlobalClassBoundaries": false,
        "ignoreKeywords": [ "FOR" ]
        },
```

**Comment by larshp** [comment]
thanks, reproduced

![image](https://user-images.githubusercontent.com/5888506/131324731-40cb3374-dd3e-4fb3-bf77-6db05bdec13c.png)

**Comment by larshp** [comment]
@OberstVonGatow try again, 2.77.3 published

depending on what you are using, it might not be updated yet

docker images update in an hour or so, vscode extension not updated yet, currently updating abaplint.app

**Comment by OberstVonGatow** [comment]
Thanks for the quick fix 👍
